docs/examples/website-change.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: Check for Website Change
+nav_order: 8
+parent: Examples
+---
+
+# Check if Website has changed Example
+
+This example demonstrates a use case for a hypothetical scenario when you need to periodically check if a website has changed. It makes use of workflow variables to keep persistent data to test against.
+
+Most websites would likely have minor changes if you were to just fetch their HTML. These changes are usually stuff like time and session properties generated on each request. We don't want to categorize these changes as true as these are false positives. Luckily many websites have one or both header values: `Etag` and `Last-Modified`. If either one of these headers changes it is safe to assume that the website has been updated. Note: Although many websites have adopted this standard, there are still websites that have not. These websites are not supported.
+
+This example is fairly simple and can be broken up into four steps:
+1. Fetch the current headers of the target website.
+2. Get the saved headers variables from the previous execution of this workflow.
+3. Compare the current and previous headers to determine if the target website has changed.
+4. Save current headers to workflow variables.
+
+Below we'll explain each step in more detail. For this example we've chosen `https://docs.direktiv.io` as the target website.
+
+## Fetch Website Headers
+Here we are calling a HTTP `POST` request on the target website using our `vorteil/request:v5` direktiv app as a function. Since we only care about the `Last-Modified` and `Etag` headers we then extract those values and store them in the `lastModified` and `etag` properties using the transform field in the `fetch-site-headers` state. Now that we have fetched the current header values required, the state will transition to the `get-old-headers` state.
```

Comment:
>  Here we are calling a HTTP `POST` request

I think it's more correct to say that you are performing a request, rather than calling it.